docs/resources/discovery.md

```diff
@@ -40,25 +40,19 @@ Resource for managing discovery on OpenManage Enterprise.
 
 ### Optional
 
-- `community_types_string` (Boolean) - Enable the use of SNMP community strings to receive SNMP traps using Application Settings in OpenManage Enterprise. 
+- `community_string` (Boolean) - Enable the use of SNMP community strings to receive SNMP traps using Application Settings in OpenManage Enterprise. 
```

Comment:
I think the name "community_string" for a boolean value is unintuitive. Please check if the name can be made better.

Reply:
does renaming it to "enable_community_strings" will be intuitive?

Reply:
Yes, sounds good